### PR TITLE
GW-56: Develop upcoming events in events page

### PR DESCRIPTION
**Pull request by Julian-UBC**
- Created a short landing message in events page
- Modify the events layout to accommodate events page's design 

Example when there are multiple upcoming events: 
Screenshot 2023-05-04 at 02-46-45 GISAUBC – Gado-Gado Indonesian Students Association of UBC

### CODE REVIEW

File: src/shared/layout/MainContainer.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from "react";
 
-export const MainContainer = ({ children }) => {
-  return <div className="w-4/5 mx-auto">{children}</div>;
+export const MainContainer = ({ className, children }) => {
+  return <div className={`${className} w-4/5 mx-auto`}>{children}</div>;
```
Comment (chrishadrian) [inline src/shared/layout/MainContainer.js:4]: Can you replace the className with the literal value of `mt-20` here? Container like this shouldn't have dynamic styles. Also adjust the other pages once you put mt-20 here.

File: src/shared/layout/event-layout/EventLayout.js [hunk 2]
```diff
@@ -33,33 +34,49 @@ export const EventLayout = ({
...
-        </div>
-        {!isMobileView && (
-          <div className="lg:w-[95%]">
-            <p className="text-center text-xl xl:text-2xl font-bold">
-              {event.title}
-            </p>
-            {event.desc.map((text) => (
-              <p className="text-center">{text}</p>
-            ))}
+    content = (
```
Comment (chrishadrian) [inline src/shared/layout/event-layout/EventLayout.js:37]: Not to be rude but can you revert this? I already got a cleaner responsive grid layout for this. Or you can just leave it and I will adjust it in my PR.

File: src/shared/layout/event-layout/EventLayout.js [hunk 3]
```diff
@@ -68,18 +85,20 @@ export const EventLayout = ({
...
-          events.length === 0 ? "mt-12" : "mt-4"
-        } `}
-      >
-        <Button
-          text={button2.name}
-          transparentBg={true}
-          handleClickButton={handleClickButton2}
-        />
-      </div>
+      <div className={`${isEventsPage ? `mt-16` : `mt-10`}`}>{content}</div>
```
Comment (chrishadrian) [inline src/shared/layout/event-layout/EventLayout.js:88]: This too

File: src/pages/events/Events.js [hunk 4]
```diff
@@ -3,13 +3,32 @@ import { Box } from "@mui/material";
...
         <EventLandingImage />
-        <MainContainer></MainContainer>
+        <MainContainer className={`mt-20`}>
+          <span
+            className={`font-proxima-nova text-primary text-3xl sm:text-4xl xl:text-5xl font-bold`}
+          >
+            Connect with fellow Indonesians at our socials and large-scale
+            events.
+          </span>
+          <EventLayout
```
Comment (chrishadrian) [inline src/pages/events/Events.js:21]: This too

**Review by chrishadrian — CHANGES_REQUESTED** [review]

**Review by gisauweb — APPROVED** [review]

**Comment by chrishadrian** [comment]
This PR is included in another PR (https://github.com/gisauweb/GISAUBC/pull/51).

**Pull request closed without merging**